src/core/dbus-manager.c

```diff
@@ -168,9 +169,18 @@ static int property_set_log_level(
         if (r < 0)
                 return r;
 
+        if (isempty(t)) {
+                log_set_max_level(m->default_log_max_level);
+                m->log_max_level_locked = false;
```

Comment:
is "locked" really the right word? Maybe use something like "override"?

Reply:
you meant "overriden" ?